go.mod

```diff
@@ -88,11 +87,11 @@ require (
 	github.com/redis/go-redis/v9 v9.0.5 // indirect
 	github.com/tidwall/match v1.1.1 // indirect
 	github.com/tidwall/pretty v1.2.1 // indirect
-	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc v1.28.0 // indirect
-	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp v1.28.0 // indirect
-	go.opentelemetry.io/otel/metric v1.32.0 // indirect
-	go.opentelemetry.io/otel/sdk/metric v1.32.0 // indirect
-	go.opentelemetry.io/proto/otlp v1.3.1 // indirect
+	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc v1.28.0
+	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp v1.28.0
+	go.opentelemetry.io/otel/metric v1.32.0
+	go.opentelemetry.io/otel/sdk/metric v1.32.0
+	go.opentelemetry.io/proto/otlp v1.3.1
```

Comment:
Can we move them out from the indirect block, please? :pray:

Reply:
I've moved, but if we really care about it (e.g. personally I never manually edit/see it, only with tooling) we need to find a way (linter) to automate this check

Reply:
I think it's easier to identify issues by a quick look if the block is consistent, definitely not a must to have, but still something nice.
Unfortunately, I don't think a linter exists for it, it would require us to write one.